auth/user_mgt_test.go

```diff
@@ -157,6 +158,267 @@ func TestInvalidGetUser(t *testing.T) {
 	}
 }
 
+// Checks to see if the users list contain the given uids. Order is ignored.
+//
+// Behaviour is undefined if there are duplicate entries in either of the
+// slices.
+//
+// This function is identical to the one in integration/auth/user_mgt_test.go
+func sameUsers(users [](*UserRecord), uids []string) bool {
+	if len(users) != len(uids) {
+		return false
+	}
+
+	sort.Slice(users, func(i, j int) bool {
+		return users[i].UID < users[j].UID
+	})
+	sort.Slice(uids, func(i, j int) bool {
+		return uids[i] < uids[j]
+	})
+
+	for i := range users {
+		if users[i].UID != uids[i] {
+			return false
+		}
+	}
+
+	return true
+}
+
+func TestGetUsers(t *testing.T) {
+	t.Run("should be rejected when given more than 100 identifiers", func(t *testing.T) {
```

Comment:
Each of these `t.Run` calls should be a top-level test case. Just give them descriptive names:

```
TestGetUsersExceeds100
TestGetUsersEmpty
TestGetUsersAllNonExisting
etc...
```

Sub tests are useful when you want to share some state between two or more test cases.